modules/ROOT/pages/java-support.adoc

```diff
@@ -91,30 +91,26 @@ If you want to change the cipher's default values, see xref:mule-runtime::tls-co
 
-The most commonly used Anypoint Connectors and modules are Java 17-compatible now, and others are being released regularly. For a list of Java 17-compatible connectors, see https://help.salesforce.com/s/articleView?id=000782248&type=1[Java 17-Compatible Anypoint Connectors^].
+=== Where to Find Java Compatibility Information for Connectors and Modules
 
-To upgrade Anypoint Connectors:
+The most commonly used Anypoint Connectors and modules are Java 17-compatible now, and others are being released regularly. 
 
-. Update your Mule app to run on Mule runtime 4.6 or later. 
-. Change the target JVM at the project property level. This includes both upgrades and downgrades of the JVM version.
-. In Anypoint Studio, update the connectors in your app to the latest Java 17-compatible version. 
-. xref:mule-runtime::deploying.adoc[Deploy your applications] to a target running on Mule 4.6 or later.
+Java compatibility information is provided on the details page for connectors and modules in the *Asset versions* table. This is the most current compatibility information, as connectors are tagged as Java 17-compatible when they are released. 
 
-[[anypoint-studio]]
-== Anypoint Studio
+For a list of Java 17-compatible connectors, see https://help.salesforce.com/s/articleView?id=000782248&type=1[Java 17-Compatible Anypoint Connectors^]. 
 
-Starting with Anypoint Studio (Studio) 7.17, if your Mule app is running on Mule runtime 4.6 or later, you can change the target JVM at the Studio project level to upgrade or downgrade the JVM. 
+=== Upgrading Anypoint Connectors 
 
-Studio 17 provides compatibility guidance:
+To upgrade Anypoint Connectors provided by MuleSoft, with no custom code:
 
-* When you compile your app, Studio 7.17 offers real-time guidance and alerts you if there are any incompatible connector versions or project mismatches to prevent deployment failures.
-* When you add a new connector or module to your project, Studio shows the supported JVM version for each version of the connector or module. 
-* When you upgrade an existing Studio project to Java 17, Studio automatically searches Exchange and suggests which extensions (modules and connectors) in your app to upgrade. 
-* When you deploy an app to CloudHub from Studio, Runtime Manager proactively detects mismatches between the project's Java version and the deployment environment. For example, if your project is built for Java 8 and your target environment is Java 17, Runtime Manager and Studio provide guidance to prevent deployment failures.
+. Update your Mule app to run on Mule runtime 4.6 or later. 
+. Change the target JVM at the project property level. This includes both upgrades and downgrades of the JVM version.
+. In <<anypoint-studio>>, update the connectors in your app to the latest Java 17-compatible version. 
```

Comment:
Studio automatically offers to update all connectors to the latest version when a project is open. There's also a button to explicitly tell Studio to do that. I would make that explicitly and link to whatever article describes that flow. 

Also notice that keeping connectors up to date is a good practice in general regardless of the Java update. As a matter of fact, changing the JVM is not necessary for Studio to auto upgrade all connectors